brushfire-core/src/main/scala/com/stripe/brushfire/Dispatched.scala

```diff
@@ -63,17 +63,18 @@ object Dispatched {
     splits.map { split =>
       new Split[Dispatched[A, B, C, D], T] {
         def predicates = split.predicates.map {
-          case (pred, p) => (wrapPredicate(pred)(fn), p)
+          case (pred, p) => (wrapPredicate(fn)(pred), p)
         }
       }
     }
   }
 
-  def wrapPredicate[X, A: Ordering, B, C: Ordering, D](predicate: Predicate[X])(fn: X => Dispatched[A, B, C, D]): Predicate[Dispatched[A, B, C, D]] =
+  def wrapPredicate[X, A: Ordering, B, C: Ordering, D](fn: X => Dispatched[A, B, C, D])(predicate: Predicate[X]): Predicate[Dispatched[A, B, C, D]] =
```

Comment:
I generally prefer fn arguments to go at the end (Ruby-style). I realize you're doing this to allow the currying below, but I'm not sure that's really a win. I guess we should try to come up with a lightweight style guide to address such issues.